src/views/checkups/network/components/form/CheckupsNetworkFormNodes.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { Dispatch, SetStateAction, SyntheticEvent, useMemo, useState } from 'react';
+
+import { modelToGroupVersionKind, NodeModel } from '@kubevirt-ui/kubevirt-api/console';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Button,
+  ButtonVariant,
+  Checkbox,
+  Dropdown,
+  DropdownItem,
+  DropdownToggle,
+  Flex,
+  FormGroup,
+  Popover,
+  PopoverPosition,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+
+import { V1Node } from '../../utils/types';
+
+type CheckupsNetworkFormNodes = {
+  isNodes: boolean;
+  nodeSource: string;
+  nodeTarget: string;
+  setIsNodes: SetStateAction<boolean>;
+  setNodeSource: Dispatch<SetStateAction<string>>;
+  setNodeTarget: Dispatch<SetStateAction<string>>;
+};
+
+const CheckupsNetworkFormNodes = ({
+  isNodes,
```

Comment:
This name doesn't make sense to me. Could we rename it to something else?

Reply:
isNodeChecked?
isNodesSelected?
sure way not.. pick something :)